security-groups/main.tf

```diff
@@ -77,4 +77,9 @@ locals {
     "${data.terraform_remote_state.natgateway.natgateway_common-nat-public-ip-az2}/32",
     "${data.terraform_remote_state.natgateway.natgateway_common-nat-public-ip-az3}/32",
   ]
+
```

Comment:
This needs to be enclosed in a "locals" block.
https://www.terraform.io/docs/configuration/locals.html